### ISSUE TITLE
[Bug] 3.7 randomly crashing and update not installing..

### ISSUE THREAD

**Issue by xinwolfthorn** [henrypp/simplewall#1500]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.7

### Windows version

windows 10

### Steps to reproduce

run 3.7 on windows 10, after a while an error message of "the instruction at 0x %p referenced memory at 0x %p. the memory could not be %s. (0xC0000005)".

### Actual behavior

it has tried and i have tried through the program update menu to update the program to 3.7.1 but when it asks to install the update i hit ok and it freezes for a second and then that dialog goes away with no update done.

**Comment by xinwolfthorn** [comment]
forgot dump files [https://www.dropbox.com/scl/fo/c4j2isdhoijsby7x6nhme/h?rlkey=609rzdblvwd40ywebqgysyrk7&dl=0](url)

**Comment by henrypp** [comment]
>  I have used the search function to see if someone else has already submitted the same bug report.

aga, #1487

**Comment by xinwolfthorn** [comment]
I don't believe this is the same issue, I never had windows defender chime in and complain about anything.
